synapse/models/inet.py

```diff
@@ -453,6 +453,138 @@ def _revModl201709181501(self):
         for prop in delprops:
             self.core.delRowsByProp(prop)
 
+    @modelrev('inet', 201709271521)
+    def _revModl201709271521(self):
+        '''
+        Rename inet:net* to inet:web*
+        '''
+
+        darks = []
+        forms = [
+            ('inet:netuser', 'inet:web:acct'),
+            ('inet:netgroup', 'inet:web:group'),
+            ('inet:netmemb', 'inet:web:memb'),
+            ('inet:follows', 'inet:web:follows'),
+            ('inet:netpost', 'inet:web:post'),
+            ('inet:netfile', 'inet:web:file'),
+        ]
+        props = [
+            ('inet:netuser:site', 'inet:web:acct:site'),
+            ('inet:netuser:user', 'inet:web:acct:user'),
+            ('inet:netuser:dob', 'inet:web:acct:dob'),
+            ('inet:netuser:url', 'inet:web:acct:url'),
+            ('inet:netuser:webpage', 'inet:web:acct:webpage'),
+            ('inet:netuser:avatar', 'inet:web:acct:avatar'),
```

Comment:
Per chat discussion, can we add inet:web:acct:loc as a string value for a user's self-declared location?

Reply:
yes

Reply:
added

Reply:
(I don't need to migrate it because it didn't exist before)